src/porklock/validation.clj

```diff
@@ -45,16 +45,24 @@
     (throw+ {:error_code ERR_PATH_NOT_ABSOLUTE
              :path (:destination options)}))
 
-  (if-not (:config options)
-    (throw+ {:error_code ERR_MISSING_OPTION
-             :option "--config"}))
+  (when-not (:debug-config options)
```

Comment:
These validations might conflict if --debug-config is set, so they're skipped in that case.